### ISSUE TITLE
Fat user interface elements

### ISSUE THREAD

**Issue by crushcafeteria** [evolus/pencil#180]
* Operating system: Ubuntu 16.04
* Pencil version: 3.0.0-rc-1
I have been designing applications with Pencil for some time. It is my favorite sketcher. One thing I loved with the older versions was just how easy the UI was. Nothing was bigger than it needed to be and everything was clear with emphasised black text. The move to Electron is just genius. Next we need to work on making it the cutest and most usable opensource sketching app. I will dedicate some of my time to this goal.

Here is what im thinking:-

![git](https://cloud.githubusercontent.com/assets/1842480/19718841/ce97bc8a-9b6f-11e6-914b-09c1f37e9e13.png)

**Comment by TeHMoroS** [comment]
I must agree here. The UI feels like there's a lot of space wasted (not a good thing for software that's used for prototyping UIs ;) ). CrushCafeteria has a point with the proposed changes. I get a feeling however, that it was designed with large screen sizes in mind. Still, on 1600x900 it feels, well, too big.

I'm also wondering if "My Shapes" and "Clipart" could be positioned on the right bar, as I have a feeling, that they won't be used as much as the main toolbox. That way the "Shapes" tab naturally dissapears, leaving only the toolbox on the left (with the vertical tabs made a bit slimmer).

**Comment by denysvitali** [comment]
This is related to #173

**Comment by altbrooks** [comment]
I do not like reading sideways text and it is sometimes hard to find collections.  Can there be an option to view them horizontally or vertically in the "settings" to accommodate all users?
